### ISSUE TITLE
False positive with Spring @Import annotations

### ISSUE THREAD

**Issue by remy-tiitre** [nebula-plugins/gradle-lint-plugin#120]
Gradle Lint reports:
warning   unused-dependency                  this dependency is unused and can be removed
backend/build.gradle:70
compile 'io.springfox:springfox-bean-validators:2.6.1'

But in reality it is used at compile time like this:
import springfox.bean.validators.configuration.BeanValidatorPluginsConfiguration;
...
@Configuration
@EnableSwagger2
@Import(BeanValidatorPluginsConfiguration.class)
public class SwaggerConfig {

and initially it was like this:
@Configuration
@EnableSwagger2
@Import(springfox.bean.validators.configuration.BeanValidatorPluginsConfiguration.class)
public class SwaggerConfig {

So question is, does gradle lint know how to handle Spring @Import annotations?

**Comment by dbolger** [comment]
Was also having this happen - this can be fixed by overriding the `visitArray` method in the DependencyAnnotationVisitor:
```java
@Override
AnnotationVisitor visitArray(String name) {
    return this
}
```
This will subsequently call the `readType` method, determining the type, and calling `readObjectName` to find any references in use.
